packages/components/src/components/SignIn.vue

```diff
@@ -250,6 +250,10 @@ export default {
   padding: 1.5rem;
```

Comment:
This looks unrelated to this particular change, but I'm noticing that the `height: 100%` on `.signin-sidebar` will cause text to overflow the container if the view is too narrow.
![image](https://github.com/user-attachments/assets/87474700-cf8c-41d8-bd31-d0184390fa63)